Apps/Src/SendTritium.c

```diff
@@ -646,7 +646,7 @@ void Task_SendTritium(void *p_arg){
         state.stateHandler();    // do what the current state does
         #ifndef SENDTRITIUM_EXPOSE_VARS
         readInputs();   // read inputs from the system
-        UpdateDisplay_SetAccel(accelPedalPercent);
+        UpdateDisplay_SetBackEMF(accelPedalPercent, brakePedalPercent);
```

Comment:
is there a reason that you only care abt the backemf and not the accel percent? Just curious